File: generators/app/templates/middle-tier/src/main/groovy/spring-mvc/controller/VersionController.java
package <%= packageName %>.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * An example web service API controller.
 */
@RestController
public class VersionController {

    @GetMapping(value = "/api/version")
    public String version() {
        return "{\"version\": \"1.0.0\"}";
    }

}

File: generators/app/templates/middle-tier/src/main/kotlin/spring-mvc/controller/VersionController.kt
package <%= packageName %>.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

/**
 * An example web service API controller.
 */
@RestController
class VersionController {

    @GetMapping(value = ["/api/version"])
    fun version(): String {
        return "{\"version\": \"1.0.0\"}"
    }

}
